nilearn/datasets/atlas.py

```diff
@@ -139,13 +139,13 @@ def fetch_atlas_difumo(dimension=64, resolution_mm=2, data_dir=None,
 
     fdescr = _get_dataset_descr(dataset_name)
 
-    params = dict(description=fdescr, maps=files_[1], labels=labels)
-
-    return Bunch(**params)
+    return Bunch(description=fdescr, maps=files_[1], labels=labels)
 
 
 @fill_doc
-def fetch_atlas_craddock_2012(data_dir=None, url=None, resume=True, verbose=1):
+def fetch_atlas_craddock_2012(homogeneity='spatial', grp_mean=True,
```

Comment:
Why not put the new parameters at the end of the signature ?

Reply:
Thank for pointing this our @bthirion, it's fixed now +1